### DISCUSSION TITLE
El País

### DISCUSSION THREAD

**Discussion by audiamus** [audiamus/AaxAudioConverter#53] [Show and tell]
AAA Audio Converter has been featured in a lifestyle column in the renowned Spanish paper _El País_:
https://cincodias.elpais.com/cincodias/2020/12/23/lifestyle/1608726945_497807.html

**Comment by Sailfish** [comment]
Congrats! Well deserved recognition.

-- 
Best Regards,
Pat

From: audiamus 
Sent: Friday, January 29, 2021 9:19 AM
To: audiamus/AaxAudioConverter 
Cc: Subscribed 
Subject: [audiamus/AaxAudioConverter] El País (#53)

AAA Audio Converter has been featured in a lifestyle column in the renowned Spanish paper El País:
https://cincodias.elpais.com/cincodias/2020/12/23/lifestyle/1608726945_497807.html
